conf/messages.en

```diff
@@ -591,7 +591,7 @@ declaration.procedureCodes.additionalProcedureCode.mandatory.error = Enter an ad
 declaration.procedureCodes.additionalProcedureCode.maximumAmount.error = Maximum amount of additional procedure codes added
 declaration.procedureCodes.additionalProcedureCode.empty = Enter an additional procedure code in the correct format
 declaration.procedureCodes.additionalProcedureCode.duplication = Additional procedure code already added
-declaration.procedureCodes.additionalProcedureCode.remove.hint = Remove additional code {0}
+declaration.procedureCodes.additionalProcedureCode.remove.hint = additional code {0}
```

Comment:
Those removes and changes are added automatically?

Reply:
The screen-reader reads the all the text in the <a> tag - this includes the contents of the visually hidden span.  E.g.
```
<a class="govuk-link govuk-link--no-visited-state" href="/customs-declare-exports/declaration/other-party/0.2114720731/remove" target="_self">Remove<span class="govuk-visually-hidden">party Consolidator GB213321323123.</span></a>
```
will be announced as "Remove party Consolidator...."